util/util-internal-dump-cli/src/prometheus.ts

```diff
@@ -56,7 +58,21 @@ export class PrometheusServer {
                     kind: 'failure'
                 }, metrics.connectionErrors)
             }
-        });
+        })
+
+        this.s3RequestsGauge = new Gauge({
```

Comment:
It should be counter

Reply:
Probably Guage suits here better. We call `set` instead of `inc` because we get exact value of metric from `S3Fs.getMetrics`. Incremet will require to calculate increase or change S3Fs significantly.

Reply:
We won't be able to count requests normally on restarts with gauge, which is always going to start from 0